### ISSUE TITLE
Is it work with videos without extensions? 

### ISSUE THREAD

**Issue by ingvilow** [mylisabox/flutter_mjpeg#28]
I have a live stream which is without extensions and I am trying to use you package (because my video is MJPEG format) but I am getting error: "Invalid codec"

**Comment by jaumard** [comment]
Yes it doesn't care about extensions, just the stream data. There is no  "Invalid codec" message in this library so not sure how I can help without more info
